File: redsheet-portal/src/app/database-models/v-lite-their-team-member.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteTheirTeamMember {
    constructor(vLiteTheirTeamMember?: vLiteTheirTeamMember )  {
        let properties = [ "liteProjectBackgroundId", "liteTheirTeamMemberId", "liteTheirTeamMemberName", "personalityId", "personalityKeyText", "position",];
        for (let property of properties)
            if (vLiteTheirTeamMember && vLiteTheirTeamMember[property])
                this[property] = vLiteTheirTeamMember[property];
    }
 
	liteProjectBackgroundId : number =   0 ;
 
    @range(1,2147483647)
	liteTheirTeamMemberId : number =   undefined;
 
    @required()
    @maxLength(500)
	liteTheirTeamMemberName : string =   undefined;
 
    @range(1,2147483647)
	personalityId : number =   undefined;
 
    @maxLength(10)
	personalityKeyText : string =   undefined;
 
    @required()
    @maxLength(500)
	position : string =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteTheirTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteTheirTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class vLiteTheirTeamMember  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int LiteProjectBackgroundId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LiteTheirTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string LiteTheirTeamMemberName { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int PersonalityId { get; set; } 
 
	[MaxLength(10)]
	public string PersonalityKeyText { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string Position { get; set; } 

	public vLiteTheirTeamMember()
	{
    }

    }
}

